state/validation.go

```diff
@@ -89,7 +89,7 @@ func (s *State) ValidatorLoop() {
 		if r := recover(); r != nil {
 			s.EventService.EmitNodeErrorMessage(eventmessages.NodeMessageCode_GENERAL,
 				"A panic state occurred in ValidatorLoop.", r)
-
+			fmt.Println(r)
```

Comment:
Any reason to include it in the grant release?

Reply:
Without it an incorrect grant activation height (which must be divisible by 25 and then added 1) would cause factomd to shut down without telling you the reason why, which was really confusing when I was testing. I included it because it's a minor change directly related to the grant activation height, but that can also go in a separate commit.